packages/web-react/src/components/Tabs/README.md

```diff
@@ -0,0 +1,90 @@
+# Tabs
+
+The Tab component is used to organize related content. User can navigate between
+groups of information in tabbable regions.
+
+## Tab
+
+```jsx
+<Tabs defaultSelectedTab={1}>
+  <Tab>
+    <TabItem forTab={1}>Item Selected</TabItem>
+    <TabItem forTab={2}>Item</TabItem>
+  </Tab>
+  <TabContent>
+    <TabPane tabId={1}>Pane 1</TabPane>
+    <TabPane tabId={2}>Pane 2</TabPane>
+  </TabContent>
+</Tabs>
+```
+
+## Tab with Links
+
+```jsx
+<Tabs defaultSelectedTab={1}>
+  <Tab>
+    <TabItem forTab={1}>Item Selected</TabItem>
+    <TabItem forTab={2}>Item</TabItem>
+    <TabItem href="https://www.example.com">Item Link</TabItem>
+    <TabItem href="https://www.example.com">Item Link Only Desktop</TabItem>
+  </Tab>
+  <TabContent>
+    <TabPane tabId={1}>Pane 1</TabPane>
+    <TabPane tabId={2}>Pane 2</TabPane>
+  </TabContent>
+</Tabs>
+```
+
+### Tabs
+
+**Available props**
+
+| Name                 | Type               | Description                            |
+| -------------------- | ------------------ | -------------------------------------- |
+| `defaultSelectedTab` | `string`, `number` | Identification of default selected tab |
+| `children`           | `any`              | Child component                        |
+
+### Tab
+
+Tab list
+
+**Available props**
+
+| Name       | Type  | Description     |
+| ---------- | ----- | --------------- |
+| `children` | `any` | Child component |
+
+### TabItem
+
+Tab list item
+
+**Available props**
+
+| Name       | Type               | Description           |
+| ---------- | ------------------ | --------------------- |
+| `forTab`   | `string`, `number` | Identification of tab |
+| `href`     | `string`           | External link         |
```

Comment:
Shouldn't we note these two options are mutually exclusive? Looking into the implementation, I am even considering making it two separate components…